src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
+  <div data-qa="notification" class="success">${message}</div>
+  `);
+};
+
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
```

Comment:
```suggestion
  body.insertAdjacentHTML('beforeend', `
```
notification content should be in body